Assets/Scripts/Waypoint Scripts/WaypointProperties.cs

```diff
@@ -134,8 +146,15 @@ public void SetLine()
 
                 endpoint = prevPoint.transform.position;
                 LineProperties.SetPosition(1, endpoint);
-                LineProperties.startWidth = controller.GetComponent<MapInteractions>().actualScale.y / 200;
-                LineProperties.endWidth = controller.GetComponent<MapInteractions>().actualScale.y / 200;
+
+                // Code in WaypointSystemUpgrade
+                ComputeWorldScaleActual();
+                LineProperties.startWidth = WorldScaleActual.y / 200;
```

Comment:
What is LineProperties and what does this do? Where did the 200 come from?

Reply:
It's what makes a line appear between waypoints. 200 is arbitrary, I think ISAACS v1 had it there from the beginning.